docs/benchmarks/test-results-summary.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Performance Benchmarks and Test Reports
+---
+
+
+## Introduction
+This page summarizes key performance metrics for nwaku and provides links to detailed test reports.
+
+> ## Quick reference
+>
+> - **10%** decrease in bandwidth usage in larger networks (1000 vs 2000 relay nodes)
```

Comment:
I think the actual bandwidth values are what's interesting here:
- libp2p bandwidth usage fluctuates between `min` and `max` kbps for topologies of up to 1000 nodes, with average bandwidth usage at `avg` kbps.
- this bandwidth decreases to an average of `avg_2` for larger topologies of.... + why?

Reply:
Mmm. I think in fact we should mention that it remains roughly _the same_ and that this is expected for Relay networks. Slight fluctuation could be due to simulation artifacts or chance differences in routing/connectivity between test runs.